src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -538,7 +583,7 @@ private static Expression BindParameterFromValue(ParameterInfo parameter, Expres
 
             var argument = Expression.Variable(parameter.ParameterType, $"{parameter.Name}_local");
 
-            if (parameter.ParameterType == typeof(string))
+            if (parameter.ParameterType == typeof(string) || parameter.ParameterType == typeof(IFormFile))
```

Comment:
This doesn't seem right. What's this for?

Reply:
Without this the special case for `typeof(string)` was skipped, and then it would throw an `InvalidOperationException` due to `IFormFile` not having a `TryParseMethod()`.

This was just enough to get the tests passing re-using `BindParameterFromValue()`, I wasn't sure what the best way to have it handle the parameter was so I just went with this pending feedback on what the preferred way would be.

I guess a custom method similar to `BindParameterFromService()`?

Reply:
1663381e146e825cea31e4ec86e6bdab06d508d6